plugins/destination/postgresql/cloud-config-ui/e2e-tests/validate-config.spec.ts

```diff
@@ -4,12 +4,12 @@ import YAML from 'yaml';
 
 test('Submit the form', async ({ page }) => {
   await page.goto('/');
-  await page.getByLabel('Host *').click();
```

Comment:
It looks like `*` used to mean required, but now means secret.

Reply:
yeah, this is on my list too, but added here for visibility: https://github.com/cloudquery/cloudquery-issues/issues/2486